src/dotnet/OpenAPI.Validator.Tests/Resource/OpenAPI/Validation/body-top-level-properties.json

```diff
@@ -79,10 +79,14 @@
             "modelAsString": false
           }
         },
-        "location": {
+        "Location": {
```

Comment:
do we still need to make this one and "Type" non compliant with the other rule? could we just leave "eTag" in the example for testing?